packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -157,6 +170,58 @@ export default class ChildProcessWorker implements WorkerInterface {
     this._resolveExitPromise();
   }
 
+  private async monitorHeartbeatError() {
+    let error;
+
+    if (this._options.inspector !== undefined) {
```

Comment:
flip `if-else` around so `true` case is first

Reply:
`true` case is already first, no ?

Reply:
I mean check `===` first, not `!==`. Essentially https://eslint.org/docs/rules/no-negated-condition 🙂